### PR TITLE
Update token reward modal

### PR DESCRIPTION
**Pull request by nkrulikovsky**
- Update Token claim modal with MUI Dialog (Only replaced Modal with MUI Dialog. Will remove scss and replace all component with MUI on another branch) .
 
##  Context
https://github.com/saddle-finance/saddle-frontend/issues/782

## Screenshot
Screen Shot 2022-01-12 at 5 22 10 PM
Screen Shot 2022-01-12 at 5 22 28 PM

### CODE REVIEW

File: cypress/integration/topMenu.test.ts [hunk 1]
```diff
@@ -11,17 +11,17 @@ context("Top menu test", () => {
...
+  it("open token claim dialog", () => {
     cy.get("[data-testid=rewardButton]").should("exist").click()
 
     cy.get("[data-testid=tknClaimContainer]")
       .should("exist")
       .find("[data-testid=tokenAddBtn]")
       .click()
     // cy.get("[data-testid=claimsListContainer]").find("Button").first().click()
-    cy.get("[data-testid=modalContainer]")
+    cy.get("[data-testid=tokenClaimDialog]")
```
Comment (ug02fast) [inline cypress/integration/topMenu.test.ts:22]: good name 👍

File: src/components/TokenClaimDialog.tsx [hunk 2]
```diff
@@ -77,105 +86,124 @@ export default function TokenClaimModal(): ReactElement {
...
+    <Dialog
+      open={open}
+      scroll="body"
+      onClose={onClose}
+      sx={{ overflow: "hidden" }}
+      data-testid="tokenClaimDialog"
+    >
+      {/* TODO: Remove this button after update the modal */}
+      <IconButton
+        disableRipple
```
Comment (ug02fast) [inline src/components/TokenClaimDialog.tsx:98]: lol, is there a way to globally disable ripples?

File: src/components/TokenClaimDialog.tsx [hunk 3]
```diff
@@ -27,7 +28,15 @@ import useAddTokenToMetamask from "../hooks/useAddTokenToMetamask"
 import { useRetroMerkleData } from "../hooks/useRetroMerkleData"
 
 // TODO: update token launch link
-export default function TokenClaimModal(): ReactElement {
+
+interface TokenClaimModalProps {
```
[the remark this answers is not in the document]
Comment (nkrulikovsky) [reply src/components/TokenClaimDialog.tsx:32]: 🙏🏻

File: src/components/TopMenu.tsx [hunk 4]
```diff
@@ -6,12 +6,12 @@ import React, { ReactElement, useContext, useRef, useState } from "react"
 
 import Button from "./Button"
 import { IS_SDL_LIVE } from "../constants"
-import Modal from "./Modal"
+// import Modal from "./Modal"
```
Comment (ug02fast) [inline src/components/TopMenu.tsx:9]: del

**Review by ug02fast — CHANGES_REQUESTED** [review]
Looks good just one line needs to be deleted.

Does it makes sense to leverage mUI Box component over divs as described here: https://stackoverflow.com/a/62521728 ?

File: src/components/TokenClaimDialog.tsx (hunk 2 above)
Comment (nkrulikovsky) [reply src/components/TokenClaimDialog.tsx:98]: There is .
I did this on the another branch. But it is in progress. :)

Comment (nkrulikovsky) [reply src/components/TokenClaimDialog.tsx:98]: We can. 
I already added that on another branch and that branch is still in progress. Let me remove that props when we update modal again.

File: src/components/TopMenu.tsx (hunk 4 above)
Comment (nkrulikovsky) [reply src/components/TopMenu.tsx]: removed!

File: cypress/integration/topMenu.test.ts (hunk 1 above)
Comment (nkrulikovsky) [reply cypress/integration/topMenu.test.ts:22]: 🙏🏻

**Comment by nkrulikovsky** [comment]
Thank you for sharing the link related MUI box.
Yes. In my case,  it is very convenient to give css style as props instead of using classes, especially on large codebase.

**Review by ug02fast — APPROVED** [review]

**Review by hammeiam — APPROVED** [review]
Looks great! Love to see more MUI usage

**Pull request merged**